src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -514,7 +514,7 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			Command command = new Command(modElement);
 			command.commandName = modElement.getName();
 			command.permissionLevel = getRandomItem(random, new String[] { "No requirement", "1", "2", "3", "4" });
-			command.onCommandExecuted = new Procedure("procedure2");
+			command.argsxml = "<xml><block type=\"args_start\" deletable=\"!_true\"></block></xml>";

```

Comment:
As you do separate tests for commands, you can remove command from here entirely, the same as it is with procedure mod element